README.md

```diff
@@ -1,11 +1,21 @@
 ![](./resources/official_armmbed_example_badge.png)
 # Thread statistics Mbed OS example
 
-This guide reviews the steps required to get thread statistics on an Mbed OS enabled platform.
+The example project is part of the Arm Mbed OS Official Examples and is the getting started example for Mbed OS. It contains an application that demonstrates how to generate thread statistics in Mbed OS.
 
-You can build this project with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command-line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli):
+You can build the project with all supported Mbed OS build tools. However, this example project specifically refers to the command-line interface tool Arm Mbed CLI. (Note: To see a rendered example you can import into the Arm Online Compiler, please see our import quick start.)
```

Comment:
The online compiler import quick start in the Mbed docs is for blinky, so that could be removed. It might also help to have a link to both mbed-cli and mbed-tools here, and to the Mbed docs build tools page.
```suggestion
You can build the project with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command-line interface tools, [Arm Mbed CLI 1](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli) and [Mbed CLI 2](https://github.com/ARMmbed/mbed-tools#installation).
```